src/main/java/wolox/training/models/Book.java

```diff
@@ -110,6 +117,7 @@ public String getTitle() {
     }
 
     public void setTitle(String title) {
+        Preconditions.checkNotNull(title, "title must be not null");
```

Comment:
Add the following validations:

- That the field is not empty

Reply:
Change constant